OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -870,6 +882,18 @@ void Manager::initialize(SimTK::State& s, double dt )
 }
 //_____________________________________________________________________________
 /**
+* set and initialize a SimTK::TimeStepper
+*/
+void Manager::initializeTimeStepper(
+    const SimTK::System& sys, const SimTK::State& state)
```

Comment:
Yeah, I don't like using this inconsistency to slip in a CMCActuatorSystem. 
Where it is being used is in:
```cpp
void VectorFunctionForActuators::
evaluate( const SimTK::State& s, double *aX, double *rF)
```
And in this case I believe the `_model` is completely unnecessary. This may be easily replaced with using a TimeStepper directly. Here to the Manager is being created and destroyed over and over for every `dt` interval during CMC. Funny thing is a Manager is used but analyses and reporting have to be turned off since these live a model whose associate with the Manager but now being integrated. It is quite horrible.